pageserver/src/tenant/timeline.rs

```diff
@@ -1298,6 +1298,10 @@ impl Timeline {
     }
 }
 
+/// Number of times we will compute partition within a checkpoint distance. Currently, this is set to 10, which
+/// indicates that we will recompute partition 10 times within a checkpoint distance.
```

Comment:
```suggestion
/// Number of times we will compute partition within a checkpoint distance.
```

I don't think we need to expand the current value in a comment, it's not material to what the constant represents, it just creates maintenance burden and the example does not really add any more information.